src/components/ServiceDetails/AboutService/ServiceDemo/Purchase/ExpiredSession/MetamaskFlow/index.js

```diff
@@ -207,7 +207,10 @@ class MetamaskFlow extends Component {
     return this.PaymentInfoCardData.find(el => el.title === "Channel Balance").value;
   };
 
-  shouldContinueBeEnabled = () => this.state.mpeBal > 0 && this.props.isServiceAvailable;
+  shouldContinueBeEnabled = () => {
+    const { channelBalance, totalPrice } = this.state;
+    return channelBalance >= totalPrice;
```

Comment:
where are we setting the channelBalance?
Actually, mpeBal should be same as channelBalance.

Reply:
After fetching available balance from channel we are setting channel balance to local variable